Created new documentation for the new Ambient Weather platform compon… (#7641)
home-assistant/home-assistant.github.io: source/_components/weather.ambient.markdown

---
layout: page

> title: "Ambient"

Reviewer: ```suggestion
title: "Ambient Weather"
```

> description: "Instructions on how to integrate Ambient weather network within Home Assistant."

Reviewer: ```suggestion
description: "Instructions on how to integrate Ambient Weather Network within Home Assistant."
```

date: 2018-11-24 09:00
sidebar: true
comments: false
sharing: true
footer: true
logo: ambient-weather-logo.png
ha_category: Weather

> ha_release: 0.32

Reviewer: This seems a little low... 😉

ha_iot_class: "Cloud Polling"
---

> The `ambient` weather platform uses [Ambient Weather](https://ambientweather.net/) as a source for information from a local weather station. To use this platform you must have an Ambient weather station installed at your location and the station must support sending weather data over the Internet to Ambient.

Reviewer: ```suggestion
The `ambient` weather platform uses [Ambient Weather](https://ambientweather.net/) as a source for information from a local weather station. To use this platform you must have an Ambient Weather station installed at your location and the station must support sending weather data over the Internet to Ambient Weather.
```

## {% linkable_title Configuration %}

You need an API key which is free but requires a [registration](https://ambientweather.net/).

> To add ambientweather to your installation, add the following to your `configuration.yaml` file:

Reviewer: ```suggestion
To add Ambient Weather to your installation, add the following to your `configuration.yaml` file:
```

```yaml
# Example configuration.yaml entry
weather:
  - platform: ambient
    api_key: YOUR_API_KEY
```

{% configuration %}
api_key:

>   description: Your API key for [Ambient](https://ambientweather.net/).

Reviewer: ```suggestion
  description: Your API key for [Ambient Weather](https://ambientweather.net/).
```
